code/src/index.css

```
@@ -1,13 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap');
```

I will just highlight what Johanna pointed out for me in one of my projects.

Good practice to add box-sizing: border box at the top. 

This is what she suggested to add to the the top of the css-file. Must be honest and say that I don't fully understand the after and before thing.

```suggestion
*,
*::after,
*::before {
  box-sizing: border-box;
}

@import url('https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap');
```